src/js/render_wordle.js

```diff
@@ -116,6 +118,89 @@ inverted_wordles.makeLayout = function (instance) {
     return layout;
 };
 
+/** Return the computed value of the font size of the given DOM element.
+ * @param {Object} elm - A DOM element
+ * @return {Number} The computed value of the font size of the given element.
+ */
+inverted_wordles.getFontSizeValue = function (elm) {
+    const style = window.getComputedStyle(elm, null).getPropertyValue("font-size");
+    return parseFloat(style);
+};
+
+/** Return the number at the midpoint if length is odd, otherwise the average of the two middle numbers.
+ * See: https://www.w3resource.com/javascript-exercises/fundamental/javascript-fundamental-exercise-88.php
+ * @param {Array} numArray - An array of numbers
+ * @return {Number} The median value of values in an array
+ */
+inverted_wordles.getMedianFontSize = function (numArray) {
+    const mid = Math.floor(numArray.length / 2), nums = [...numArray].sort((a, b) => a - b);
+    return numArray.length % 2 !== 0 ? nums[mid] : (nums[mid - 1] + nums[mid]) / 2;
+};
+
+/** Finds font sizes of all wordle texts and calculate these numbers:
+ * 1. The media number of all font sizes;
+ * 2. The gap number between min and median font sizes;
+ * 3. The gap number between media and max font sizes.
+ * @param {Object} instance - The singleton instance. Calculated numbers are returned by attaching to it.
+ */
+inverted_wordles.calSizeGaps = function (instance) {
```

Comment:
This has a funny name and I think it would be clearer if it just computed the minimum, median, and maximum font sizes. Perhaps calculateFontSizes?
Also it would be better if it accepted the array of text elements in its arguments rather than computing it as a side effect.